src/transformStateWithClones.js

```diff
@@ -7,7 +7,35 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const states = [];
+  const copyState = { ...state };
```

Comment:
```suggestion
  const stateCopy = { ...state };
```

Reply:
Just FYI - copyState - is an action, stateCopy - is a result of this action (copied state)